packages/react-components/src/__tests__/utils.test.tsx

```diff
@@ -19,36 +19,42 @@ describe('getSvgAspectRatio', () => {
 
 describe('isInternalLink', () => {
   describe.each`
-    description          | href
-    ${'external link'}   | ${`https://parkinsonsroadmap.org/`}
-    ${'external link'}   | ${`//parkinsonsroadmap.org/`}
-    ${'external link'}   | ${`//${window.location.hostname}:${Number(window.location.port || 80) - 1}/`}
-    ${'external mailto'} | ${`mailto:test@${window.location.hostname}`}
-  `('for an $href to be an $description', ({ href }) => {
-    it('returns false', () => {
-      expect(isInternalLink(href)).toBe(false);
+    description          | href                                                                          | absolute
+    ${'external link'}   | ${`https://parkinsonsroadmap.org/`}                                           | ${`https://parkinsonsroadmap.org/`}
+    ${'external link'}   | ${`//parkinsonsroadmap.org/`}                                                 | ${`http://parkinsonsroadmap.org/`}
+    ${'external link'}   | ${`//${window.location.hostname}:${Number(window.location.port || 80) - 1}/`} | ${`http://localhost:79/`}
+    ${'external mailto'} | ${`mailto:test@${window.location.hostname}`}                                  | ${`mailto:test@localhost`}
+  `('for an $href to be an $description', ({ href, absolute }) => {
+    it('returns false and absolute url', () => {
+      const [internal, absoluteUrl] = isInternalLink(href);
+      expect(internal).toBe(false);
+      expect(absoluteUrl).toEqual(absolute);
     });
   });
 
-  describe.each([
-    `${window.location.protocol}//${window.location.host}`,
-    `${window.location.protocol}//${window.location.host}/page`,
-    `//${window.location.host}`,
-    `//${window.location.host}/page`,
-    `/`,
-    `/page`,
-    `.`,
-    `./page`,
-    `..`,
-    `../page`,
-    `page`,
-    `#`,
-    `#fragment`,
-    `?query`,
-    `${window.location.protocol}//${window.location.host}/page?query#fragment`,
-  ])('for an internal link with a router to %s', (href: string) => {
-    it('returns true', () => {
-      expect(isInternalLink(href)).toBe(true);
+  describe.each`
+    href                                                                          | relative
```

Comment:
I think `relative` is confusing here, they're still absolute paths, just without an origin. Maybe `stripped`?

Reply:
And for `absolute` instead `full`?